infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/pvc/PVCSubPathHelper.java

```diff
@@ -154,6 +186,17 @@ void execute(
       String pvcName,
       String[] commandBase,
       String... arguments) {
+    execute(null, workspaceId, namespace, pvcName, commandBase, Collections.emptyMap(), arguments);
```

Comment:
Is this `execute` function just for test cases? If so, doesn't it make sense to just remove it and make the actual function below visible for testing? It's strange to have what seems to be a wrapper for setting `identity=null`.

Reply:
no, it's called from `removeDirsAsync`, which is async operation where we don't have the identity.